deployment-examples/gcp/extension/k8s/demo/secrets.yaml

```diff
@@ -1,10 +0,0 @@
-sensitiveEnvs:
```

Comment:
Why we are removing this file in this branch, I guess this will be needed ?  Same for notification secrets.yaml files, also in docs they are dead links now.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/3dda2cf61b60c6789485168fb8f135d29fe8f723